remoteit/desktop
# [DESK-567] Sidebar remote context display

### Description


Show a sidebar in remote view to provide context on what device they are currently viewing or managing.

### Checklist




### Test plan



1. Connect to a remote device on LAN
2. Sign in 
3. See sidebar display connected to device through lan
4. Connect to a remote device through remote.it
5. See sidebar display connected to device through remtoe network
6. Connect locally at localhost:29999 Do not see remote sidebar

### Ticket



<https://remoteit.atlassian.net/browse/DESK-567>
